apps-rendering/src/components/headlineTag.tsx

```diff
@@ -0,0 +1,28 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { background } from '@guardian/common-rendering/src/editorialPalette/background';
+import { text } from '@guardian/common-rendering/src/editorialPalette/text';
+import type { ArticleFormat } from '@guardian/libs';
+import { headline, remSpace } from '@guardian/source-foundations';
+import type { ReactElement } from 'react';
+
+const headlineTagStyles = (format: ArticleFormat): SerializedStyles => css`
+	background-color: ${background.headlineTag(format)};
+	color: ${text.headlineTag(format)};
+	${headline.xxsmall({ fontWeight: 'bold', lineHeight: 'loose' })}
+	display: inline-block;
+	box-decoration-break: clone;
```

Comment:
Maintains the padding if the tag spans across multiple lines, otherwise it only applies to the first line. 
Arguably we can get rid of it because the word `Interview` before the headline is the only instance in AR where we use this component but if we ever want to reuse it and apply longer tags, then it's helpful to keep this styling. Happy to remove it though, not fussed either way.